tests/test-link-preview.md

```diff
@@ -4,20 +4,74 @@ Link previews should embed previews of the contents of a hyperlink from a messag
 
 Post location variation: 
 
-1. Post as message in center channel with RHS closed (link preview should render under message) 
```

Comment:
Updated with expected on same line, and shortened sentence so instructions remain clear and compact.